.github/workflows/build_all.yml

```diff
@@ -70,6 +70,14 @@ jobs:
         with:
           fetch-depth: 0
 
+      # Install a 32-bit Python so building related stuff work.
+      - name: Setup x86 Python
+        if: matrix.platform == 'x86'
```

Comment:
Would make more sense to do on both IMO.

Reply:
The x64 version already has a properly setup Python, and a different version: less homogeneous configurations mean more chances to catch bugs.

Reply:
Fair enough,